Adding a link to more info on Android Drawables (#9844)
flutter/website: src/platform-integration/android/splash-screen.md

Android provides the concept of a [launch screen][] to
display a `Drawable` while the app is initializing.

A `Drawable` is an Android graphic.

> For information on adding a `Drawable` to your

Reviewer: **nit:** Make this an action.
```suggestion
To learn how to add a `Drawable` to your
```

</style>
```

> [drawables]: {{site.android-dev}}/studio/write/resource-manager#import

Reviewer 2: That link seems to talk about how to import Drawables into an Android project using Android Studio.

Maybe https://developer.android.com/guide/topics/resources/drawable-resource is a better URL to send developers to?

Author: I saw that page, too. I thought maybe adding it to the project was needed? I mean, a Drawable itself is pretty normal.

Reviewer 2: Ahh, Okiday
